examples/text-editor.roc

```diff
@@ -192,14 +189,10 @@ getFileContentsTask = \{ fileExists, path } ->
 # Get the file path from the first argument
 readArgFilePath : Task.Task Path.Path _
 readArgFilePath =
-    args <- Arg.list |> Task.attempt
-
+    args <- Arg.list {} |> Task.attempt
```

Comment:
We should use `!` here if we can -- backpacking is deprecated.

```roc 
args = Arg.list! {}
```

Though I have a vague memory this may have been left because there was a bug.

Reply:
May I push my nix flake configuration? I'm building roc from source, but I keep getting an error when I use this last syntax,

or maybe it is something different?